integration_tests/features/StressTest.feature

```diff
@@ -0,0 +1,35 @@
+Feature: Stress Test
+
+@long-running
+Scenario: Stress Test
+    Given I have a seed node NODE1
+    And I have a seed node NODE2
+    And I have 1 base nodes connected to all seed nodes
+    And I have wallet WALLET_A connected to seed node NODE1
+    And I have wallet WALLET_B connected to seed node NODE2
+    And I have a merge mining proxy PROXY connected to NODE1 and WALLET_A
+    # We need to ensure the coinbase lock heights are gone; mine enough blocks
+    When I merge mine 8 blocks via PROXY
+    Then all nodes are at height 8
+    When I wait for wallet WALLET_A to have at least 15100000000 tari
+    # A coin split can produce at most 499 outputs
+    And I coin split tari in wallet WALLET_A to produce 499 UTXOs of 7000 uT each with fee_per_gram 20 uT
+    And I coin split tari in wallet WALLET_A to produce 499 UTXOs of 7000 uT each with fee_per_gram 20 uT
+    # Make sure enough blocks are mined for the coin split transaction to be confirmed
+    When I merge mine 6 blocks via PROXY
+    Then all nodes are at height 14
+    Then wallet WALLET_A detects all transactions as Mined_Confirmed
+    When I wait 1 seconds
+    When I send 400 transactions of 1111 uT each from wallet WALLET_A to wallet WALLET_B at fee_per_gram 20
```

Comment:
I would add a transaction broadcast confirmation step here before continuing with the test:
```suggestion
    When I send 400 transactions of 1111 uT each from wallet WALLET_A to wallet WALLET_B at fee_per_gram 20
    Then all wallets detect all transactions are at least Broadcast
```

Reply:
This issue is handled by the fact that while I am waiting for transaction to be confirmed I am also mining. I felt that is a much more realistic situation when you consider how the real network would pump transactions in the system.

Reply:
That is true, but from a test resiliency viewpoint, this step may be necessary, perhaps with a `# TODO: Remove this detect broadcast step when the system is more resilient` comment?  In my current test I have been waiting 3800 s for that step to complete for the `When I send 1400 transactions` test case and it did not finish.

Reply:
I would argue that the purpose of this test is not for it to "finish" successfully, it's to be able to produce a stress test that is close to a real world configuration of the network and reproduce the issues we see when we do a stress test on the main network to as to debug how it breaks.

As such I wouldn't want to add steps that take this further away from how the real network behaves in order to achieve a passing test.